cmd/mdatagen/loader.go

```diff
@@ -226,6 +230,8 @@ type metadata struct {
 	ShortFolderName string `mapstructure:"-"`
 
 	Tests *tests `mapstructure:"tests"`
+
+	Config *config `mapstructure:"config"`
```

Comment:
I find the 

```
config:
  generate: true
```

confusing.

My suggestion would be something like

```
config_documentation:
   enabled: true
```

or even

```
config:
  documentation:
     enabled: true
```

where it is clear that we are talking about the configuration documentation and not about generating the configuration structs themselves.

Reply:
sounds good to me!